src/functions-reference/binary_distributions.Rmd

```diff
@@ -147,64 +158,98 @@ dropping constant additive terms.
 
 `real` **`bernoulli_logit_glm_lpmf`**`(int y | matrix x, real alpha, vector beta)`<br>\newline
 The log Bernoulli probability mass of y given chance of success
-`inv_logit(alpha + x * beta)`, where the same intercept `alpha` and dependant variable value `y` are used
-for all observations. The number of columns of `x` needs to match the size of the
-coefficient vector `beta`.
-If `x` and `y` are data (not parameters) this function can be executed on a GPU.
+`inv_logit(alpha + x * beta)`.
+
+If `x` and `y` are data, this function can be executed on a GPU.
```

Comment:
@rok-cesnovar we can cut this line out now right?

Reply:
Indeed we can yes.

Reply:
Should I remove them for all distributions? (please say yes)

Reply:
Sir yes sir!

Reply:
Sir I will remove all of these messages sir!

Reply:
Messages removed